design/workitems.go

```diff
@@ -138,6 +138,7 @@ var _ = a.Resource("workitem", func() {
 			a.Param("filter[workitemtype]", d.UUID, "ID of work item type to filter work items by")
 			a.Param("filter[area]", d.String, "AreaID to filter work items")
 			a.Param("filter[workitemstate]", d.String, "work item state to filter work items by")
+			a.Param("filter[category]", d.UUID, "work item type category to filter work items by")
```

Comment:
@hectorj2f I used the term "work item type category" to differentiate between `work item type category` and `work item link type category`